app/javascript/components/document_adder.vue

```diff
@@ -53,14 +56,22 @@ export default {
       if (recording.file_set_ids.length < 1) {
         return
       }
+	// Ensure that the new FileSet IDs are unique
+	const new_file_set_ids = recording.file_set_ids.filter( file_set_id => !this.member_file_set_ids.includes(file_set_id) )
```

Comment:
Rather than handling this here, can we just make the controller action not add them twice? Could even be in another PR.